lib/fathead/cppreference_doc/README.md

```diff
@@ -1,19 +1,23 @@
-Cppreference fathead plugin
+Cppreference Fathead
 ===========================
 
-This is a fathead plugin for C++ standard library reference manual hosted at
-http://en.cppreference.com.
+This is a Fathead for C++ standard library reference manual hosted at http://en.cppreference.com.
 
-Data source
+Data Source
 -----------
 
 fetch.sh downloads and untars a snapshot of the website, which is available at
-http://en.cppreference.com/w/Cppreference:Archives. A new snapshot is released
-periodically. The location of the newest archive that is tested for DuckDuckGo
-is upload.cppreference.com/w/Cppreference:DDG-link?action=raw.
+http://en.cppreference.com/w/Cppreference:Archives. A new snapshot is released periodically.
+
+The location of the newest archive that is tested for DuckDuckGo is upload.cppreference.com/w/Cppreference:DDG-link?action=raw.
```

Comment:
@rasikapohankar let's hyperlink this URL as well. Starting the URL with `http://` will hyperlink it in markdown.